packages/supi/src/install/index.ts

```diff
@@ -314,6 +314,9 @@ export async function mutateModules (
           break
         }
         case 'unlinkSome': {
+          if (project.manifest?.name) {
+            await rimraf(path.join(opts.globalBin, project.manifest?.name))
```

Comment:
This is not enough.

1. a package may have several different bins
2. every bin may have a `ps1` file, a `cmd` file and a `bash` file

Here you can see how bins are removed from node_modules/.bin

https://github.com/pnpm/pnpm/blob/394902ac62ee14bedd7df6d42e8cd28891eb1123/packages/modules-cleaner/src/removeDirectDependency.ts#L62-L85

maybe a new package should be created like `@pnpm/remove-bins`